### DISCUSSION TITLE
Is it acceptable to add non-executable binary into Homebrew-core?

### DISCUSSION THREAD

**Discussion by Misaka13514** [Homebrew/discussions#1424] [Everyday usage] [answered]
I want to add some resource files shared by multiple formulas to homebrew-core.

- Homebrew/homebrew-core/pull/72660
- Homebrew/homebrew-core/pull/73155
- Homebrew/homebrew-core/pull/74955
- Homebrew/homebrew-core/pull/76466

These resources files are now used in:
- v2ray
- xray
- trojan-go (currently not merged)

**Comment by SMillerDev** [comment, marked as answer]
Yeah, there's tons of libraries in homebrew-core. It just needs to be open source, it doesn't need to be executable.

**Reply by Misaka13514** [reply]
v2fly/geoip requires some resources (`GeoLite2-Country-CSV.zip` from [MaxMind](https://dev.maxmind.com/geoip/geoip2/geolite2/) ), and these resources are difficult to download.
Is binary release file acceptable? 😢 The problem becomes more complicated.
Or should I create a cask

**Reply by SMillerDev** [reply]
> Is binary release file acceptable?

If it's precompiled executables, no. But if it's just some binary data I don't see an issue with it.
